orm/model/ormdb/module.go

```diff
@@ -119,12 +120,12 @@ func NewModuleDB(schema *ormv1alpha1.ModuleSchemaDescriptor, options ModuleDBOpt
 		}
 
 		db.filesById[id] = fdSchema
-		for name, table := range fdSchema.tablesByName {
+		for _, name := range maps.Keys(fdSchema.tablesByName) {
 			if _, ok := db.tablesByName[name]; ok {
 				return nil, ormerrors.UnexpectedError.Wrapf("duplicate table %s", name)
 			}
 
-			db.tablesByName[name] = table
+			db.tablesByName[name] = fdSchema.tablesByName[name]
 		}
```

Comment:
This code isn't sorting the keys, so AFAICT doesn't produce behavior any different than ranging over the map directly?